superset-frontend/src/views/CRUD/data/dataset/AddDatasetModal.tsx

```diff
@@ -105,12 +100,7 @@ const DatasetModal: FunctionComponent<DatasetModalProps> = ({
       onHide={onHide}
       primaryButtonName={t('Add')}
       show={show}
-      title={
-        <>
-          <StyledIcon name="warning-solid" />
-          {t('Add dataset')}
-        </>
-      }
+      title={<>{t('Add dataset')}</>}
```

Comment:
Is the fragment needed here? Can't we just pass a plain string for the title?

Reply:
oh you know @geido , they don't need to be a fragment. Just passed in a regular string. Thank you!